src/argus_htmx/tailwindtheme/styles.css

```diff
@@ -1,3 +1 @@
 @import "tw.css";
```

Comment:
Perhaps you want to inline the contents of tw.css into here, given that all it's doing is including tailwind